music21/expressions.py

```diff
@@ -1075,23 +1524,100 @@ def name(self) -> str:
         'turn'
 
         >>> from music21.common.enums import OrnamentDelay
-        >>> delayedInvertedTurn = expressions.InvertedTurn(delay=OrnamentDelay.DEFAULT_DELAY)
+        >>> delayedInvertedTurn = expressions.InvertedTurn(
+        ...     delay=OrnamentDelay.DEFAULT_DELAY,
+        ...     upperAccid='sharp',
+        ...     lowerAccid='natural'
+        ... )
         >>> delayedInvertedTurn.name
-        'delayed inverted turn'
+        'delayed inverted turn (upper=sharp, lower=natural)'
 
-        >>> delayedBy1Turn = expressions.Turn(delay=1.0)
+        >>> delayedBy1Turn = expressions.Turn(delay=1.0, lowerAccid='double-flat')
         >>> delayedBy1Turn.name
-        'delayed(delayQL=1.0) turn'
+        'delayed(delayQL=1.0) turn (lower=double-flat)'
 
         '''
-        superName: str = super().name
+        theName: str = super().name
         if self.delay == OrnamentDelay.DEFAULT_DELAY:
-            return 'delayed ' + superName
+            theName = 'delayed ' + theName
         elif isinstance(self.delay, (float, Fraction)):
-            return f'delayed(delayQL={self.delay}) ' + superName
-        return superName
+            theName = f'delayed(delayQL={self.delay}) ' + theName
+
+        if self.upperAccid is not None or self.lowerAccid is not None:
+            theName += ' ('
+            if self.upperAccid is not None:
+                theName += 'upper=' + self.upperAccid
+                if self.lowerAccid:
+                    theName += ', '
+            if self.lowerAccid is not None:
+                theName += 'lower=' + self.lowerAccid
+            theName += ')'
+
+        return theName
 
-    def realize(self, srcObj: 'music21.note.Note', *, inPlace=False):
+    def getUpperSize(
+        self,
+        srcPitch: pitch.Pitch,
+    ) -> interval.IntervalBase:
```

Comment:
I don't like the duplication of getUpperSize/getLowerSize -- why not a keyword to "getSize" or whatever we called it above.

Reply:
Done.